src/index.js

```diff
@@ -0,0 +1,20 @@
+const express = require("express");
+
+const routes = require("./routes");
+
+const app = express();
+
+const PORT = process.env.PORT || 3000;
+
+// express middlewares
+app.use(express.static("public"));
+
+app.use(express.urlencoded({ extended: true }));
+
+app.use(express.json({ extended: true }));
+
+app.use(routes);
+
+app.listen(PORT, () =>
+  console.log(`server running on https://localhost:${PORT}`)
```

Comment:
This is good practice for when you're a dev, but please remove from the live application